### PR TITLE
Turn off Functional tests in Travis for now.

### PR DESCRIPTION
**Pull request by ClifHouck**
Bypasses #77.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+# always pass locally. Needs investigation into issue #77. 
 env:
-  - TOX_ENV=py27
+  - TOX_ENV=travis_py27
   - TOX_ENV=pep8
 install:
   - pip install tox
 script:
-  - tox -vv -e $TOX_ENV
+  - tox -vv -e $TOX_ENV
```
Comment (jayofdoom) [inline .travis.yml]: nit: added some trailing whitespace here

**Comment by jayofdoom** [comment]
+1, fix the nit, or don't.

File: .travis.yml (hunk 1 above)
Comment (jimrollenhagen) [inline .travis.yml]: trailing whitespace :/

**Comment by jimrollenhagen** [comment]
What jay said :D

**Comment by ClifHouck** [comment]
Done. And merging since everything else stayed the same.

**Pull request merged**